jOOL-java-8/src/main/java/org/jooq/lambda/tuple/Range.java

```diff
@@ -97,4 +102,38 @@ public boolean overlaps(T t1, T t2) {
     public Optional<Range<T>> intersect(T t1, T t2) {
         return intersect(new Range<>(t1, t2));
     }
+
+    /**
+     * Whether the given value is included in the range. Assumes range inclusiveness.
+     * <p>
+     * <code><pre>
+     * // true
+     * range(1, 3).contains(1)
+     * range(2, null).contains(5)
+     *
+     * // false
+     * range(1, 3).contains(4)
+     * range(2, null).contains(1)
+     * </pre></code>
+     */
+    public boolean contains(T t) {
+        return Tuple2.contains(this, t);
+    }
+
+    /**
+     * Whether the given right tuple is included in the left tuple.
+     * <p>
+     * <code><pre>
+     * // true
+     * range(1, 3).contains(range(1, 2))
+     * range(2, null).contains(range(3, 5))
+     *
+     * // false
+     * range(1, 3).contains(range(null, 2))
+     * range(2, null).contains(range(1, 3))
+     * </pre></code>
+     */
+    public boolean contains(Tuple2<T, T> other) {
+        return Tuple2.contains(this, other);
+    }
```

Comment:
This is an entirely unrelated feature, which should be tracked and discussed in a separate issue.

Reply:
I realise that this implementation was put into `Tuple2` because `overlaps()` is there. Which might have been a mistake...

Reply:
We'll deprecate these methods on `Tuple2`: https://github.com/jOOQ/jOOL/issues/355

Reply:
yes I found that a bit weird to find them in `Tuple2`, I'm glad to read you say it's a mistake.
I've added my method next to the others as to disturb the code base the less possible, and to not make architectural choices outside the scope of the PR.

Reply:
Makes sense, thanks.